.github/workflows/base_node_binaries.yml

```diff
@@ -94,15 +94,23 @@ jobs:
           echo $tagnet
           # case match is not RegEx, but wildcards/globs
           case "$tagnet" in
-            v*-pre.*) TARI_NETWORK=esme
+            v*-pre.*)
+              TARI_NETWORK=esme
+              TARI_NETWORK_DIR=testnet
```

Comment:
Intentionally not `TARI_NETWORK_DIR=esme` I take it?

Reply:
Yeah, the whole need for a different var.

It's a gotcha of the network based compilations. You currently pass a real network when compiling, and it builds a bin for any network compatible with it. So passing `esme` builds a testnet bin comptabile for all testnets, but we can't pass `testnet` because `testnet` isn't a valid network. 

But we don't want every testnet bin to sit in an `esme` directory so here we are.